src/app/views/sidebar/resource-explorer/ResourceLink.tsx

```diff
@@ -106,13 +106,6 @@ const ResourceLink = (props: IResourceLinkProps) => {
     }
 
     if (resourceLink.type === 'path') {
-      menuItems.push(
-        {
-          key: ResourceOptions.SHOW_QUERY_PARAMETERS,
```

Comment:
I think you can also remove this unused enum from the resources.ts

Reply:
Nice catch!